### PR TITLE
Adds JvmName annotations where necessary to improve interop when deve…

### PR DESCRIPTION
**Pull request by joeldudleyr3**
…loping CorDapps in java.

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/services/config/ConfigUtilities.kt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@file:JvmName("ConfigUtilities")
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/services/config/ConfigUtilities.kt]: I'm surprised you need access to ConfigUtilities in Java code. Is it `configureTestSSL` you're using?

Comment (joeldudleyr3) [inline node/src/main/kotlin/net/corda/node/services/config/ConfigUtilities.kt]: Yep, to instantiate the RPC client:

CordaRPCClient client = new CordaRPCClient(nodeAddress, ConfigUtilitiesKt.configureTestSSL());

Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/services/config/ConfigUtilities.kt]: OK, can you add a TODO saying to remove this annotation when `configureTestSSL` is moved.

Comment (joeldudleyr3) [inline node/src/main/kotlin/net/corda/node/services/config/ConfigUtilities.kt]: Sure, thanks Shams.

**Pull request merged**